### PR TITLE
Features/v2 simplifies with basic cli

### PR DESCRIPTION
**Pull request by ViktorMalesevic**
*Issue #, if available:*

*Description of changes:*
Moving helper scripts at main folder level + update to match new config files structure, and READMEs update

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,30 +4,114 @@ As enterprise businesses embrace Machine Learning (ML) across their organisation
...
+### Fifth step: cdk deploy
+
+```bash
+cd mlops-multi-account-cdk/mlops-infra/
+pip install -requirements.txt
+cdk deploy --all --profile mlops-governance
+```
+
+```bash
+cd mlops-multi-account-cdk/mlops-sm-project-template/
```
Comment (ravibhushanratnakar) [inline README.md:92]: If there are additional python modules which are used in mlops-sm-project-template then it is obvious that we should also include
pip install -r requirements.txt

File: README.md [hunk 2]
```diff
@@ -4,30 +4,114 @@ As enterprise businesses embrace Machine Learning (ML) across their organisation
...
+
+```bash
+mlops-multi-account-cdk/scripts/quick-start.sh bootstrap
+```
+
+### Fifth step: cdk deploy
+
+```bash
+cd mlops-multi-account-cdk/mlops-infra/
+pip install -requirements.txt
```
Comment (ravibhushanratnakar) [inline README.md:87]: instead of -requirements.txt, it should be like below
pip install -r requirements.txt

**Review by ravibhushanratnakar — APPROVED** [review]
some minor changes related to pip install command

**Review by fotinosk — APPROVED** [review]
Looks good

**Pull request merged**